src/test/lib/logical_query_plan/static_table_node_test.cpp

```diff
@@ -57,7 +57,7 @@ TEST_F(StaticTableNodeTest, HashingAndEqualityCheck) {
 
   EXPECT_EQ(same_static_table_node->hash(), static_table_node->hash());
   EXPECT_NE(different_static_table_node_by_definitions->hash(), static_table_node->hash());
-  EXPECT_NE(different_static_table_node_by_constraints->hash(), static_table_node->hash());
+  EXPECT_EQ(different_static_table_node_by_constraints->hash(), static_table_node->hash());
```

Comment:
Add a comment why the hash should be equal?

I think we more and more abandoned test cases that assert hashes are NOT equal (what do we know about hash collisions), so feel free to have a look at these cases.